gdscript/sidebar.gd

```diff
@@ -660,49 +656,95 @@ func update_ui(update_diff: bool = false) -> void:
 		else:
 			change_author = "Anonymous"
 
-		var change_timestamp = human_readable_timestamp(change.timestamp)
+		var item = history_tree.create_item(root)
+		history_item_count += 1
 
-		var prefix = ""
+		# set columns
+		var column_index = 0
```

Comment:
Make the column index numbers const or enum values, e.g. NAME_COLUMN, HASH_COLUMN, etc.